src/index.html

```diff
@@ -12,7 +15,132 @@
       href="styles/index.scss"
```

Comment:
You are linking to a SCSS file, which is a preprocessor file and cannot be interpreted by the browser. You should compile your SCSS files into CSS and link to the generated CSS file instead.